OpenTrons/opentrons-api
# feat(odd): prep WizardHeader and other pipette flow components for odd

closes RLIQ-335, RLIQ-336, RLIQ-337

# Overview

Preps for ODD pipette flow work in numerous areas:
1. adds ui for ODD to `WizardHeader`
2. adds ui for ODD to `WizardRequiredEquipmentList`
3. add ui for ODD to `ChoosePipette`
4.  creates `SmallButton` for ODD

Figma:
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/OT-3-On-Device-Display?node-id=7608%3A108808&t=noJbfcevyUZ76xk4-0

# Test Plan

on an ODD:
- select on the attach pipette flow, verify that the `ChoosePipette` screen with the `WizardHeader` matches designs. The text for the header is not yet updated!

<img width="997" alt="Screen Shot 2023-02-22 at 3 24 39 PM" src="https://user-images.githubusercontent.com/66035149/220750641-db669787-351a-4b06-bfdd-d588db2804aa.png">

- select on any flow and navigate to the `BeforeBeginning` page, verify that the equipment list design matches the ODD designs.

<img width="500" alt="Screen Shot 2023-02-22 at 3 24 59 PM" src="https://user-images.githubusercontent.com/66035149/220750702-37fe1135-b0f7-4ce0-9c5f-2a7059e6c230.png">

# Changelog

- `WizardHeader`, `ChoosePipette, and `WizardRequiredEquipmentList` now get the `isOnDevice` config boolean, component is updated for ODD
- `PipetteWizardFlow` also gets the `isOnDevice` config boolean and is added to the list of props `isOnDevice` that will be used a lot for the additional ODD work
- `SmallButton` is created and different css states matches figma. 
- the `constants` for the equipment list are updated from `GEN3 Pipette` to `1- or 8-Channel Pipette` and `96 Channel` to `96-Channel` and `96 Channel Mount` to `96-Channel Mount` to match the designs
- updated all affected tests 

# Review requests

- would like to verify that it is okay that I created `SmallButton`
- make sure everything looks good and works as expected as outlined in the test plan

# Risk assessment

low
